wiki/Do_you_really_want_to_ask_peppy/ru.md

```diff
@@ -1,20 +1,15 @@
----
-outdated: true
-outdated_since: d80f130d2ee1a1de26188b49560c8b6ed876fd56
----
-
 # Вы действительно хотите спросить peppy о чем-нибудь?
 
-Привет. Я ([peppy](https://osu.ppy.sh/users/2)) понимаю ваше желание связатся со мной напрямую, но я прошу вас понять: тут не место для того, чтобы просить меня о помощи или (вообще) технической поддержке. Я отделяю техническую поддержку osu! от личных сообщений, а ещё мне тяжело отвечать на все приватные сообщения. Поэтому я прошу вас выбрать один из следующих вариантов:
+Привет.  ([peppy](https://osu.ppy.sh/users/2)) понимает ваше желание лично связаться с ним, но он просит понять, что osu! **не то место**, где стоит просить его о помощи или технической поддержке в целом. Он чётко разделяет техническую поддержку osu! от личных сообщений, и ему тяжело отвечать на все личные сообщения. Поэтому он просит вас выбрать один из следующих вариантов:
```

Comment:
is there a reason the username is in brackets?

Reply:
ehhh... oops?